ui/cap-react/src/antd/forms/widgets/RichEditorWidget.js

```diff
@@ -8,9 +8,9 @@ import Toggler from "./RichEditorPreviewPlugin";
 import "react-markdown-editor-lite/lib/index.css";
 import "katex/dist/katex.min.css";
 import "markdown-it-texmath/css/texmath.css";
-import "./styles/RichEditor.css";
+import "./styles/RichEditorWidget.css";
 
-const RichEditor = (props) => {
+const RichEditorWidget = props => {
```

Comment:
lets also update the imports in `Collection.js` (https://github.com/miguelgrc/analysispreservation.cern.ch/blob/2683/ui/cap-react/src/antd/collection/Collection.js#L6)

right now it uses the grommet one, lets update